packages/allure-playwright/src/index.ts

```diff
@@ -168,16 +180,9 @@ class AllureReporter implements Reporter {
     allureTest.addLabel(LabelName.THREAD, thread);
 
     allureTest.status = statusToAllureStats(result.status, test.expectedStatus);
-    if (result.error) {
-      const message = result.error.message && stripAscii(result.error.message);
-      let trace = result.error.stack && stripAscii(result.error.stack);
-      if (trace && message && trace.startsWith(`Error: ${message}`)) {
-        trace = trace.substr(message.length + "Error: ".length);
-      }
-      allureTest.statusDetails = {
-        message,
-        trace,
-      };
+    const error = result.error;
+    if (error) {
```

Comment:
maybe its topic for totally different PR but we also have `results.error` which is array of results and we need to process all the errors in result

Reply:
different PR, yes